src/commando/core.py

```diff
@@ -1,2 +1,79 @@
-def hello_world():
-    return "Hello, world!"
+import subprocess
```

Comment:
The code design in here is pretty hacky but it gets the job done :man_shrugging: Worth trying to come up with a better design?